determined-ai/determined
# feat: open tasks in embedded task view in CLI [DET-7686]

## Description

feat: open tasks in embedded task view [DET-7686]

JupyterLab notebooks and Tensorboard are not currently being opened in the embedded task view when using the `CLI`. These changes update the `CLI` to use the embedded task view for both.




## Test Plan
Using the CLI:
1. Create a Notebook, it should open in the embedded task view.
2. Open a Notebook, it should open in the embedded task view.
3. Create a Tensorboard, it should open in the embedded task view.
4. Open a Tensorboard, it should open in the embedded task view.




## Commentary (optional)

Currently I cannot open `Tensorboard` locally so I have yet to test that.




## Checklist

See [Release Note](../docs/release-notes/README.md) for details.





[DET-7686]: https://determinedai.atlassian.net/browse/DET-7686?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[DET-1234]: https://determinedai.atlassian.net/browse/DET-1234?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
